### PR TITLE
Adding Contact form

### PR DESCRIPTION
**Pull request by ali-techworld**
**In this project we include contact form using**

- HTML 
- CSS 
- Flexbox

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -160,5 +160,22 @@ <h3 class="css">CSS</h3>
...
+    <section class="fotter-contact-form-container">
+      <section class="fotter-contact-form-text">
+        <h2 class="fotter-contact-form-heading">Contact me</h2>
+        <p class="fotter-contact-form-subheading">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+          I’d love to help with it!</p>
+        <section class="form-conatiner">
+          <form action="https://formspree.io/f/{form_id}" method="post" class="form">
+            <input type="text" name="name" required="required" maxlength="30" placeholder="Lorem Ipsum" class="input name">
+            <input type="email" name="email" required="required" placeholder="loremipsum@gmail.com" class="input email">
+            <textarea name="textarea" class="input textarea" required="required" placeholder="Write your message here.." cols="30" rows="7"></textarea>
```
Comment (Abenezer-Tilahun) [inline index.html]: - [ ] As stated in the project requirement Kindly make sure that **Message input has maxlength="500" attribute.**

File: index.html [hunk 2]
```diff
@@ -160,5 +160,22 @@ <h3 class="css">CSS</h3>
...
+            <input type="submit" value="Get in touch" class="input form-btn">
+          </form>
```
Comment (Abenezer-Tilahun) [inline index.html]: - [ ] Kindly make sure that the  **form is submitted successfully** with valid input values.When I submit the form the I got the following error , Kindly Fix it when you integrate with [formspree service](https://formspree.io/html), you didin't create a hash id for the form So Kindly Fix it.
form

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @ali-techworld  @malodev1234 , 

Great job for implemented  the some of the Project Requirement , but there are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻
✔️ No Linter errors
✔️ Correct Git Hub-Flow
✔️ Good commit messages
✔️ Professional README

almost

## Required Changes ♻️
_Check the comments below the review._


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Alusp — APPROVED** [review]
Hi @ali-techworld 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

congratulations

## Highlights

 [ ] Linters set-up ✔️
- [ ] Commit messages descriptive ✔️
- [ ] Professional GitHub Flow used ✔️
- [ ] Profesional PR Summary ✔️
- [ ] Use Form-spree :heavy_check_mark: 
- [ ] Use HTML form Validation :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**